PLATFORMS.md

```diff
@@ -0,0 +1,63 @@
+# Supported platforms
+
+This file documents the different platforms supported by `liboqs` and therefore defines three different support tiers:
+
+## Support tiers
+
+This classification is roughly based on the [rust platform support tier classification](https://doc.rust-lang.org/beta/rustc/platform-support.html):
```

Comment:
Might I suggest either a "Tier 0" or additional criteria for "Tier 1" which includes constant-time testing and documentation of CT issues? We currently have automated CT-testing for x86 but not aarch64. Or would this be viewed as out of scope for this document?

Reply:
Sounds reasonable. Would you want to add proposed text straight to this PR?

Reply:
@SWilson4 will you do this update?

Reply:
Oh never mind I see that this was done, thanks.